heroku_deploy/describe.py

```diff
@@ -0,0 +1,7 @@
+from .utils import generate_heroku_app_name, run_shell_command
+
+
+def describe(deployment_name, heroku_config=None):
+    app_name = generate_heroku_app_name(deployment_name)
+    stdout, stderr = run_shell_command(['heroku', 'apps:info', '--app', app_name])
+    print(stdout)
```

Comment:
make sure we run python format for these files.

Reply:
Ran `black` and `isort` based on what is used in BentoML's Makefile (`make format`). Should I add a Makefile to the repo as well?